pymeasure/instruments/hp/hp8116a.py

```
@@ -93,6 +93,8 @@ def __init__(self, adapter, name="Hewlett-Packard 8116A", **kwargs):
         )
         self.has_option_001 = self._check_has_option_001()
 
+    _init_comm_pairs = [(b"CST", b"x" * 87 + b' ,\r\n')]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)